File: C_Programming/Competitive_Programming/CP_011.c
////////////////////////////////////////////////////////////////////
//
//  File Name    : Display_First_N_Even_Numbers
//
//  Description  : This program accepts a number from the user
//                 and displays the first N even numbers.
//
//  Example:
//      Input  : 5
//      Output : 2 4 6 8 10
//
//      Input  : 7
//      Output : 2 4 6 8 10 12 14
//
////////////////////////////////////////////////////////////////////


#include<stdio.h>

void PrintEven(int iNo)
{
    int iCnt = 0;

    if(iNo <= 0)
    {
        return ;
    }

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("%d\t", iCnt * 2);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number : \n");
    scanf("%d", &iValue);

    PrintEven(iValue);

    return 0;
}

File: Java_Programming/Competitive_Programming/CP_011.java
//////////////////////////////////////////////////////////////////
//
//  File Name    : Check_Leap_Year
//
//  Description  : This program accepts a year and
//                 checks whether the given year is
//                 a leap year or not.
//
//  Example:
//      Input  : 2024
//      Output : Leap Year.
//
//////////////////////////////////////////////////////////////////

class Logic
{
    void checkLeapYear(int year)
    {
        if((year % 400 == 0) || ((year % 4 == 0) && (year % 100 != 0)))
        {
            System.out.println("Leap Year.");
        }
        else
        {
            System.out.println("Not a Leap year.");
        }
    }
}

class CP_011
{
    public static void main(String A[])
    {
        Logic lobj = new Logic();

        lobj.checkLeapYear(2024);
    }
}